app/scripts/lib/rpc-method-middleware/handlers/wallet-createSession/constants.ts

```diff
@@ -0,0 +1,11 @@
+export enum KnownSessionProperties {
```

Comment:
Forgot to export this from the `@metamask/chain-agnostic-permission` package, can do as a fast follow: https://github.com/MetaMask/core/pull/5522